docs/artifacts.rst

```diff
@@ -87,27 +87,27 @@ In order to simply save the weigths, we can run the following command:
     studio run --capture=~/weights:weights train_linreg.py 
 
 Flag ``--capture`` (or ``-c``) specifies that data from folder
-``~/weights`` needs to be captured continously - every minute (frequency
+``~/weights`` needs to be captured continously - every minute (the frequency
 can be changed in a config file), and at the end of the experiment. In
 the Web ui page of the experiment we now have a link to weights
 artifact. This simple script should finish almost immediately, but for
-longer running jobs upload happens every minute of a runtime (the upload
+longer running jobs upload happens every minute of runtime (the upload
 happens in a separate thread, so this should not slow down the actual
-experiment)
+experiment).
 
 Machine-independent access to the artifacts
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-So far we have been assuming that all the experiments are being run a
-local machine; and the only interaction with artifacts has been to save
+So far we have been assuming that all experiments are being run on a
+local machine, and the only interaction with artifacts has been to save
 them for posterity's sake. But what if our experiments are growing a bit
-to big to be run locally? Fortunately, studio comes with a dockerized
+too big to be run locally? Fortunately, Studio comes with a dockerized
 worker that can run your jobs on a beefy gpu server, or on a cloud
 instance (cloud management is not provided just yet). But how do we make
```

Comment:
cloud management is already there :) could you delete the ``(cloud management is not provided just yet)`` (I know I should have been the one to have done that :)